Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml

```diff
@@ -0,0 +1,1330 @@
+      playbookName: Cortex XDR - Cloud IAM User Access Investigation
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      AWS-accessKeyRemediationType:
+        simple: Disable
+      AWS-resourceRemediationType:
+        simple: Stop
+      AWS-userRemediationType:
+        simple: Revoke
+      Azure-resourceRemediationType:
+        simple: Poweroff
+      Azure-userRemediationType:
+        simple: Disable
+      GCP-accessKeyRemediationType:
+        simple: Disable
+      GCP-resourceRemediationType:
+        simple: Stop
+      GCP-userRemediationType:
+        simple: Disable
+      alert_id:
+        complex:
+          root: inputs.alert_id
+      autoAccessKeyRemediation:
+        simple: "False"
+      autoBlockIndicators:
+        simple: "False"
+      autoResourceRemediation:
+        simple: "False"
+      autoUserRemediation:
+        simple: "False"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": -1780,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: ad830609-c796-44fc-8de4-76d6ec8ce8dd
+    type: playbook
+    task:
+      id: ad830609-c796-44fc-8de4-76d6ec8ce8dd
+      version: -1
+      name: Cortex XDR - XCloud Token Theft Response
+      description: |-
+        ---
+
+        ## Cloud Token Theft Response Playbook
+
+        The **Cloud Token Theft Response Playbook** provides a structured and comprehensive flow to effectively respond to and mitigate alerts involving the theft of cloud tokens. The playbook supports AWS, GCP, and Azure and executes the following:
+
+        **Cloud Enrichment:**
+        - Enriches the involved resources
+        - Enriches the involved identities
+        - Enriches the involved IPs
+
+        **Verdict Decision Tree:**
+        - Determines the appropriate verdict based on the investigation findings
+
+        **Early Containment using the Cloud Response - Generic Playbook:**
+        - Implements early containment measures to prevent further impact
+
+        **Cloud Persistence Threat Hunting:**
+        - Conducts threat hunting activities to identify any cloud persistence techniques
+
+        **Enriching and Responding to Hunting Findings:**
+        - Performs additional enrichment and responds to the findings from threat hunting
+
```

Comment:
```suggestion
        Determines the appropriate verdict based on the investigation findings.

        **Early Containment using the Cloud Response - Generic Playbook:**
        Implements early containment measures to prevent further impact.

        **Cloud Persistence Threat Hunting:**
        Conducts threat hunting activities to identify any cloud persistence techniques.

        **Enriching and Responding to Hunting Findings:**
        Performs additional enrichment and responds to the findings from threat hunting.

```